src/components/MainView.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row } from 'reactstrap';
+import ImageManager from './ImageManager';
+
+const MainView = ({
+  heartClass, pressionBottomClass, handleSection, sices, t,
```

Comment:
Each of these should be in their own line.

Reply:
Ok.